### DISCUSSION TITLE
PRs to v5.1 are possible?

### DISCUSSION THREAD

**Discussion by WinterSilence** [twbs/bootstrap#37314] [Q&A] [answered]
Can I send PRs with some fixes from versions >= 5.2 to 5.1?

I don't plan use versions >= 5.2 because using CSS variables everywhere is slow down page loading.

**Comment by julien-deramond** [comment, marked as answer]
All the PRs for v5 need to target the `main` branch. In other words, we won't release any new 5.1.* versions.
